source/Tasks/CompressModulesWithChecksum.build.ps1

```diff
@@ -57,15 +57,22 @@ task CompressModulesWithChecksum {
 
     if ($configurationData.AllNodes -and $CurrentJobNumber -eq 1)
     {
-        $modules = Get-ModuleFromFolder -ModuleFolder $RequiredModulesDirectory | Where-Object -FilterScript {
-            Get-DscResource -Module $_.Name
-        } # Only zip up the Modules that have Exported DSC Resources
+        $previousProgressPreference = $ProgressPreference
+        $ProgressPreference = 'SilentlyContinue'
+
+        #Only zip up the Modules that have Exported DSC Resources
+        $allModules = Get-ModuleFromFolder -ModuleFolder $RequiredModulesDirectory
```

Comment:
ah, that's the one.
It would be good to double check this function Get-DscResourceFromModuleInFolder. Does it work with class based resources?

Reply:
It seems to work. JeaDsc is part of the compressed modules when using the PRs version.